build_tools/cmake/iree_copts.cmake

```diff
@@ -417,6 +417,10 @@ iree_select_compiler_opts(IREE_DEFAULT_LINKOPTS
     ${_IREE_LOGGING_LINKOPTS}
   MSVC
     "-natvis:${IREE_ROOT_DIR}/runtime/iree.natvis"
+    
```

Comment:
pre-commit wants the trailing whitespace here removed

```suggestion

```

https://github.com/iree-org/iree/actions/runs/15196683326/job/42742383752?pr=20892